canister/src/api/metrics.rs

```diff
@@ -91,6 +91,12 @@ fn encode_metrics(w: &mut MetricsEncoder<Vec<u8>>) -> std::io::Result<()> {
         w.encode_instruction_histogram(&state.metrics.get_balance_apply_unstable_blocks)?;
         w.encode_instruction_histogram(&state.metrics.get_current_fee_percentiles_total)?;
 
+        w.encode_gauge(
+            "ins_send_transaction_count",
```

Comment:
What does "ins" stand for? Also, should we add a label to differentiate between valid and invalid requests? It might be useful to know both counters.

Reply:
"ins" stands for instructions - which now that you mention it, shouldn't be a prefix for this endpoint since it isn't counting instructions. I'll remove that.

I wanted to originally have a counter for both valid and invalid requests, but we cannot actually count invalid requests since we panic in those cases.

Reply:
Ah, that makes sense. It's fine then, the part of having invalid requests was a nit.